lib/jellyfish-plugin.ts

```diff
@@ -1,12 +1,10 @@
+import { getLogger } from '@balena/jellyfish-logger';
+import type { LogContext } from '@balena/jellyfish-logger';
+import type { ContractDefinition } from '@balena/jellyfish-types/build/core';
+import type { JSONSchema6 } from 'json-schema';
```

Comment:
Do you know if there's a reason not to use the type exported from `jellyfish-types`?

Reply:
@Ereski Just tried it out locally and got the following type error:
```
Argument of type 'JsonSchema' is not assignable to parameter of type 'JSONSchema6'. Type 'false' has no properties in common with type 'JSONSchema6'.
```

I can look more into this when I get back, but thought I'd let you know.

Reply:
@Ereski Turns out `skhema.validate` requires a `JSONSchema6` typed parameter which is incompatible (see error above) with `JsonSchema` provided by `jellyfish-types`.